.gitignore

```diff
@@ -58,3 +58,5 @@ tools-resources/
 # ignore file or report generated by the plugin sync scripts
 **/operators_plugin_sync_report.txt
 **/new_operators.txt
+out/*
+user_tools/downloaded_jars/*
```

Comment:
It is better to avoid dumping temporary files everywhere and use the USER-TOOLS-TEMP folder as a way to store temporary files.